guides/user/pipeline/pipeline-templates/instantiate.md

```diff
@@ -38,7 +38,9 @@ and get it using the following command:
      “application”: “myApp”, # Set this to the app you want to create the pipeline in.
      “name”: “<pipeline name>”, # Pipeline name, remember this for the next part.
      “template”: {
-       “source”: “spinnaker://newSpelTemplate” # Reference to the pipeline template we published above. We saved it in Spinnaker, so we prefix the template id with ‘spinnaker://’. ‘http://’ and ‘file://’ prefixes are also supported.
+       “artifactAccount”: “front50ArtifactCredentials”,
```

Comment:
Can you drop a comment specifying that the `front50*` strings are static constants?

Reply:
sure!